generators/server/templates/src/test/kotlin/package/web/rest/UserResourceIT.kt.ejs

```diff
@@ -1075,7 +1067,7 @@ class UserResourceIT <% if (databaseType === 'cassandra') { %>: AbstractCassandr
          * if they test an entity which has a required relationship to the User entity.
```

Comment:
Refer this comment. This is used by the entities that we generate. So closing the PR.